src/index.ts

```diff
@@ -22,6 +24,10 @@ export const PgAggregatesPreset: GraphileConfig.Preset = {
     PgAggregatesOrderByAggregatesPlugin,
     PgAggregatesFilterRelationalAggregatesPlugin,
   ],
+
+  schema: {
+    defaultBehavior: "+aggregates +aggregates:filterBy +aggregates:orderBy",
```

Comment:
Note you do not have to do this in a separate plugin; generally it's wise to do it in the plugin in which the behavior is consumed - this may mean two of the plugins each defining their own behaviors.

Reply:
I removed the `defaultBehavior` from the `PgAggregatesPreset` and added aggregates to `entityBehaviors`:

```ts
// AddAggregateTypesPlugin.ts

schema: {
    entityBehavior: {
      pgResource: "select aggregates",
    },
```

```ts
 // FilterRelationalAggregatesPlugin.ts
 
schema: {
    entityBehavior: {
      pgResource: "aggregates:filterBy",
    },
```

```ts
 // OrderByAggregatesPlugin.ts
 
schema: {
    entityBehavior: {
      pgCodecRelation: "select aggregates:orderBy",
    },
```

Result is the same as with `defaultBehavior`:
 - Aggregates are enabled by default;
 - Aggregates aren't removed by `defaultBehavior` in the `graphile.config.ts`:
 ```ts
 const preset: GraphileConfig.Preset = {
  extends: [
    ...
    // Change the options and add/remove plugins based on your V4 configuration:
    makeV4Preset({
      defaultBehavior: "-aggregates -aggregates:filterBy -aggregates:orderBy",
      ...
 ```

Reply:
It's possible to define the behaviors in a separate plugin.
In this case user can skip this plugin and define their own default behaviors. 
But I don't like this.

Reply:
This is likely not the fault of this code. I would expect `-*` behavior to remove everything, but it doesn't; I found this out in August and it means that there's something about the behavior system that's not quite right yet. I [started work on fixing it](https://github.com/graphile/crystal/pull/1803) back in early September but had to pivot to working on other things, but I really need to get back to it.

Reply:
Also please make sure you don't already have a `preset.schema.defaultBehavior` entry; because your `makeV4Preset` version will be overwritten by it. (You shouldn't really set it in the makeV4Preset - it should be a top-level configuration.)

Reply:
Thanks
So I'll use the `entityBehavior` instead of the `defaultBehavior` in these plugins. After you fix the behavior system, I'll test the aggregates again.

Reply:
I think it's probably fine to merge this as-is, it still allows per-entity control over aggregates, and it's possible to override by adding your own plugin that adds entity behaviors - we don't need `defaultBehavior` to work for this to be mergeable IMO. If you agree, mark it ready for review and ping me :+1:

Reply:
(You might want to add a comment to the README above the `defaultBehavior` section stating that this currently doesn't work due to a bug in PostGraphile that's being worked on: https://github.com/graphile/crystal/pull/1803 )